packages/embed-react/src/storyshots.test.ts

```diff
@@ -1,3 +1,3 @@
 import initStoryshots from '@storybook/addon-storyshots'
 
-initStoryshots({ framework: 'react' })
+initStoryshots({ framework: 'react', storyNameRegex: /^Default$/ })
```

Comment:
Would it be better to disable snapshots for the specific component?
https://github.com/storybookjs/storybook/tree/main/addons/storyshots/storyshots-core#disable